src/SWP/Bundle/ContentBundle/Rule/Applicator/ArticleRuleApplicator.php

```diff
@@ -77,6 +88,10 @@ public function apply(RuleInterface $rule, RuleSubjectInterface $subject)
 
         $subject->setTemplateName($configuration[$this->supportedKeys[1]]);
 
+        if ((bool) $configuration[$this->supportedKeys[2]]) {
```

Comment:
tests will fail and tell you whats wrong anyway, but yeah I agree referring by key is less readable here tho, will change that